sources/packages/api/src/services/student-appeal/student-appeal.service.ts

```diff
@@ -140,6 +140,15 @@ export class StudentAppealService extends RecordDataModelService<StudentAppeal>
           );
         }),
       )
+      .orderBy(
+        `CASE 
+          WHEN EXISTS(${this.studentAppealRequestsService
+            .appealsByStatusQueryObject(StudentAppealStatus.Pending)
+            .getSql()}) THEN 1
```

Comment:
We do have an enum for the sort order:
![image](https://user-images.githubusercontent.com/61259237/171301355-95bd535f-68dd-4851-8ab0-68df210b8e8c.png)